app/build.gradle

```diff
@@ -0,0 +1,63 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'com.blooddonor.android'
```

Comment:
W przypadku iOSa używamy `bundleId: pl.edu.pjwstk.goodwill-ninjas.blooddonor`. Może ujednolicimy to do:

- Android `pl.edu.pjatk.goodwill-ninjas.blooddonor-android`
- iOS `pl.edu.pjatk.goodwill-ninjas.blooddonor-ios`

Reply:
Dopasuję się u mnie (lokalnie), bo już zacząłem.

Reply:
Widzę, że użyłeś `pl.edu.pjatk.goodwill_ninjas.blooddonor_android` przez [zalecenia Androida co do app module](https://developer.android.com/studio/build/configure-app-module#set-application-id). Jest teraz git, dzięki!